masterlist.yaml

```diff
@@ -649,6 +649,7 @@ plugins:
     group: *fixesResourcesGroup
     after:
       - 'Lanterns Of Skyrim - All In One - Main.esm'
+      - 'RSkyrimChildren.esm'
```

Comment:
Recommended to load `RSkyrimChildren.esm` before USSEP on RS Children mod page description.